### DISCUSSION TITLE
Adding DTMF

### DISCUSSION THREAD

**Discussion by dvxgit-tbautista** [flutterjanus/flutter_janus_client#141] [Q&A]
Has anybody had experience adding DTMF? Could you please provide some references or materials that I could look into that would help me? Thank you!

**Comment by RaviGaruda** [comment]
@dvxgit-tbautista have you found any way for sending DTMF?

**Reply by dvxgit-tbautista** [reply]
I, and my team added a WebRTC function in one of the main files. I will make time to make a PR this week.

**Reply by RaviGaruda** [reply]
Okay, Thanks it will be great.

**Reply by RaviGaruda** [reply]
hey @dvxgit-tbautista have you submit PR for sending DTMF?

**Comment by babayanta1994** [comment]
Hi. Any news about dtmf? I really need this feature
